source/includes/examples/tf-staging-prod-complete/main.tf

```diff
@@ -1,7 +1,22 @@
+# Create a Group to Assign to Project
+resource "mongodbatlas_team" "project_group" {
+  org_id = var.atlas_org_id
+  name   = var.atlas_group_name
+  usernames = [
```

Comment:
tiny style question: on line 18 in this file we compress the array to exist on only a single line. Does it make sense to do the same thing here?